### PR TITLE
Add specs to table populated class behavior

### PR DESCRIPTION
**Pull request by farias-r7**
MS-1362

- [ ] VERIFY specs pass.

### CODE REVIEW

File: dist/marionette.carpenter.css [hunk 1]
```diff
@@ -151,11 +153,11 @@
       padding-top: 250px;
       width: 100%; }
     .table-component .table-region thead {
-      background-image: -owg(linear-gradient(#fff, #dfdfdf));
-      background-image: -webkit(linear-gradient(#fff, #dfdfdf));
-      background-image: -moz(linear-gradient(#fff, #dfdfdf));
-      background-image: -o(linear-gradient(#fff, #dfdfdf));
-      background-image: linear-gradient(#fff, #dfdfdf); }
+      background-image: -owg(compact(linear-gradient(#fff, #dfdfdf), false, false, false, false, false, false, false, false, false));
```
Comment (ugohil-r7) [inline dist/marionette.carpenter.css]: can we write mixin for this, so we don't keep on repeating.

**Comment by ugohil-r7** [comment]
:ship: it 

Only one file change but the changeset doesn't justify that, why ?

File: dist/marionette.carpenter.css (hunk 1 above)
Comment (claughlin-r7) [inline dist/marionette.carpenter.css]: @ugohil-r7 this is generated css so there is no control on what it builds.

**Comment by farias-r7** [comment]
@ugohil-r7 This fixes https://github.com/rapid7/marionette.carpenter/issues/43

This PR changes the following behavior:

Before:
.populated class was suppose to be added to the table element when the ajax request was completed, right away if it was a client side static collection.

.loaded class was added when the table dom element was rendered, even if the ajax request did not complete. (It was intended to indicate the JS portion of the code has loaded)

Populated and Loaded classes got added to the table when the table was first rendered. It did not behave as intended (Regression caused by https://github.com/rapid7/marionette.carpenter/pull/31

Now:
There is no need for the .loaded class behavior. We replace .populated class with .loaded.

Server Side Collection:
The .loaded class is applied to the table dom node whenever an ajax request has completed. It is removed when an ajax request starts.

Client Side Collection:
.loaded is applied to the table on first load

**Pull request merged**